### PR TITLE
Change log for 0.10.3 release

### PR DESCRIPTION
**Pull request by mikemhenry**
## Motivation and context

Resolves #???

## Change log

```

```

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -8,6 +8,45 @@ This section lists features and improvements of note in each release.
...
+
+* Print perses version on startup by @mikemhenry in https://github.com/choderalab/perses/pull/1176
+* Support pymbar 3 & 4 by @mikemhenry in https://github.com/choderalab/perses/pull/1173
+* Store initial and final topologies for all phases -- small molecule pipeline by @ijpulidos in https://github.com/choderalab/perses/pull/1210
+
+Documentation
+^^^^^^^^^^^^^
+
+* Added MPI example with dipeptide mutation by @ijpulidos in https://github.com/choderalab/perses/pull/1228
+* Update comments in `RESTCapableHybridTopologyFactory` doc string by @zhang-ivy in https://github.com/choderalab/perses/pull/1189
```
Comment (ijpulidos) [inline docs/changelog.rst:28]: It is a minor thing but just to be consistent. In restructured Text the way to get inline monotyped/code blocks is with double ``
```suggestion
* Update comments in ``RESTCapableHybridTopologyFactory`` doc string by @zhang-ivy in https://github.com/choderalab/perses/pull/1189
```
Same applies to all the other instances.

File: docs/changelog.rst [hunk 2]
```diff
@@ -8,6 +8,45 @@ This section lists features and improvements of note in each release.
...
+
+Testing/CI/Packaging
+^^^^^^^^^^^^^^^^^^^^
+
+* Run CI on new 0.10.x branch by @mikemhenry in https://github.com/choderalab/perses/pull/1212
+* Use python executable from env by @mikemhenry in https://github.com/choderalab/perses/pull/1174
+* Remove example testing by @mikemhenry in https://github.com/choderalab/perses/pull/1214
+* CI miscellaneous fixes by @ijpulidos in https://github.com/choderalab/perses/pull/1217
+* Improve docker building by @mikemhenry in https://github.com/choderalab/perses/pull/1200
+* Clean examples -- CLI protein-ligand example for Tyk2 by @ijpulidos in https://github.com/choderalab/perses/pull/1223
```
Comment (ijpulidos) [inline docs/changelog.rst:45]: I think we could move this one to documentation, rather than testing.

**Review by ijpulidos — COMMENTED** [review]
Looks good, just a couple of minor comments. Thanks!

**Review by ijpulidos — APPROVED** [review]
LGTM!

**Comment by mikemhenry** [comment]
Cool, once it gets merged I will tag and make the release

**Pull request merged**